src/librustc/hir/intravisit.rs

```diff
@@ -210,6 +210,10 @@ pub trait Visitor<'v> : Sized {
         }
     }
 
+    fn visit_arg(&mut self, arg: &'v Arg) {
```

Comment:
Well, the proper naming would be `param`, but that would introduce some discord since the existing code around the new code already uses `arg`.
Renaming everything could be a refactoring material for a separate PR.

Reply:
Filed https://github.com/rust-lang/rust/issues/62426 for a potential separate PR.